app/src/main/java/org/oppia/android/app/home/promotedlist/PromotedStoryListViewModel.kt

```diff
@@ -0,0 +1,51 @@
+package org.oppia.android.app.home.promotedlist
+
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.ViewModel
+import org.oppia.android.R
+import org.oppia.android.app.home.HomeItemViewModel
+import org.oppia.android.app.home.RouteToRecentlyPlayedListener
+import org.oppia.android.app.shim.IntentFactoryShim
+
+/** [ViewModel] promoted story list in [HomeFragment]. */
```

Comment:
This reads a bit weird. Perhaps: ``Promoted story list view model in [HomeFragment].``?

Reply:
Oops, done